flexmeasures/data/schemas/scheduling/storage.py

```diff
@@ -109,6 +109,10 @@ class StorageFlexModelSchema(Schema):
     storage_efficiency = EfficiencyField(data_key="storage-efficiency")
     prefer_charging_sooner = fields.Bool(data_key="prefer-charging-sooner")
 
+    stock_gain = fields.List(
+        QuantityOrSensor("MWh"), data_key="stock-gain", required=False
```

Comment:
So far we haven't exposed API users to the term *stock*. Perhaps *soc-gain* makes the flex-model look less eclectic.

I also think we may split this into two fields, one for *soc-gain* and another for *soc-usage*, both strictly non-negative. This has the advantage of not having to specify usage (which I feel is the more common use case) as negative quantities or as negative sensor data.

Alternatively, we only introduce the *soc-usage* field, allowing negative values to denote a gain, and we set up the field to switch the sign when assigning the internal variable `gain` (using `unit="-MWh"`).

Reply:
Good point! I've created two fields `soc-gain` and `soc-usage`. Internally, this is converted to `stock delta` to make it more "neutral".